bin/common.sh

```diff
@@ -38,17 +38,17 @@ function abort() {
 }
 trap abort SIGINT
 
-#######################################
-# https://github.com/google/styleguide/blob/gh-pages/shellguide.md#stdout-vs-stderr
-#######################################
+#####################################################################################
+# https://github.com/google/styleguide/blob/gh-pages/shellguide.md#stdout-vs-stderr #
+#####################################################################################
```

Comment:
Broad interpretation of https://github.com/google/styleguide/blob/gh-pages/shellguide.md#s4.2-function-comments.

Don't mind to change it, my personal opinion is that this looks better.